build.sbt

```diff
@@ -160,7 +162,14 @@ lazy val core = (crossProject(JSPlatform, JVMPlatform) in file ("core"))
   )
   .jsSettings(jsOpts)
   .jsSettings(
-    prevVersions := jsPrevVersions
+    prevVersions := jsPrevVersions,
+    libraryDependencies ++= {
+      if (!isDotty.value) Seq(
+        "org.scalatest"     %%% "scalatest"       % scalatestVersion,
+        "org.scalatestplus" %%% "scalacheck-1-15" % scalatestPlusScalacheckVersion
```

Comment:
I'm unhappy with this duplication, but hopefully it's temporary.  I tried to extract these into `Dependencies.scala`, but the `%%%` only works in magical places.